pilot/pkg/serviceregistry/synthetic/serviceentry/syntheticserviceentrycontroller.go

```diff
@@ -149,8 +149,8 @@ func (c *SyntheticServiceEntryController) dispatch(config model.Config, event mo
 }
 
 // RegisterEventHandler registers a handler using the type as a key
+// Note: currently it is not called
```

Comment:
I think it is actually used [here](https://github.com/istio/istio/blob/fc099f72404973929afebab1e9fec32e9a5e5393/pilot/pkg/serviceregistry/synthetic/serviceentry/discovery.go#L68)

Reply:
Normally, it is called here
```
		for _, descriptor := range schemas.Istio {
			s.configController.RegisterEventHandler(descriptor.Type, configHandler)
		}
	}
```

However, you removed SSE previously